fvm/crypto/crypto_test.go

```diff
@@ -205,13 +228,6 @@ func TestVerifySignatureFromRuntime(t *testing.T) {
 					require.NoError(t, err)
 					require.False(t, sigOk)
 				},
-			}, {
```

Comment:
Maybe we could keep this test since it tests a random tag?

Reply:
I put it back but had to adjust to NoError and returning sigOK, which is similar to "user" case now

Reply:
yep it's similar to "user". 
"user" used to be another special tag for something (can't remember) and this was a specific test for that special tag to make things worked for it.